pkg/cloud/services/elb/loadbalancer.go

```diff
@@ -1485,6 +1485,7 @@ func fromSDKTypeToLB(v *elbv2.LoadBalancer, attrs []*elbv2.LoadBalancerAttribute
 	return res
 }
 
+// chunkELBs is similar to https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/main/pkg/cloud/services/gc/utils.go#L46.
```

Comment:
Since this is the same project we should probably point to the relative path of the file?
Also, maybe since lines can change, copy at least the function name here or something so we find what this was pointing to in the future. :)

Reply:
How about something like:
# chunkELBs is similar to chunkResources in pkg/cloud/services/gc